python/doc/examples/reliability_sensitivity/sensitivity_analysis/plot_sensitivity_wingweight.py

```diff
@@ -0,0 +1,462 @@
+
+# %%
+pcc_indices = corr_analysis.computePCC()
+print(pcc_indices)
+
+# %%
+#
+
+# %%
+graph = ot.SobolIndicesAlgorithm.DrawCorrelationCoefficients(
+    pcc_indices, inputNames, "PCC coefficients - Wing weight")
+view = otv.View(graph)
+
+# %%
+# PRCC coefficients
+# -----------------
+# We compute here `PRCC` coefficients using the `CorrelationAnalysis`.
+
+# %%
+prcc_indices = corr_analysis.computePRCC()
+print(prcc_indices)
+
+# %%
+graph = ot.SobolIndicesAlgorithm.DrawCorrelationCoefficients(
+    prcc_indices, inputNames, "PRCC coefficients - Wing weight")
+view = otv.View(graph)
+
+# %%
+# SRC coefficients
+# -------------------
+# We compute here `SRC` coefficients using the `CorrelationAnalysis`.
+
+# %%
+src_indices = corr_analysis.computeSRC()
+print(src_indices)
+
+# %%
+graph = ot.SobolIndicesAlgorithm.DrawCorrelationCoefficients(
+    src_indices, inputNames, 'SRC coefficients - Wing weight')
+view = otv.View(graph)
+
+# %%
+# Normalized squared SRC coefficients (coefficients are made to sum to 1) :
+
+# %%
+squared_src_indices = corr_analysis.computeSquaredSRC(True)
+print(squared_src_indices)
+
+# %%
+# And its associated graph:
+
+# %%
+graph = ot.SobolIndicesAlgorithm.DrawCorrelationCoefficients(
+    squared_src_indices, inputNames, 'Squared SRC coefficients - Wing weight')
+view = otv.View(graph)
+
+# %%
+#
+
+# %%
+# SRRC coefficients
+# --------------------
+# We compute here `SRRC` coefficients using the `CorrelationAnalysis`.
+
+# %%
+srrc_indices = corr_analysis.computeSRRC()
+print(srrc_indices)
+
+# %%
+graph = ot.SobolIndicesAlgorithm.DrawCorrelationCoefficients(
+    srrc_indices, inputNames, 'SRRC coefficients - Wing weight')
+view = otv.View(graph)
+
+# %%
+# Pearson coefficients
+# -----------------------
+# We compute here the Pearson :math:`\rho` coefficients using the `CorrelationAnalysis`.
+
+# %%
+pearson_correlation = corr_analysis.computePearsonCorrelation()
+print(pearson_correlation)
+
+# %%
+graph = ot.SobolIndicesAlgorithm.DrawCorrelationCoefficients(pearson_correlation,
+                                                             inputNames,
+                                                             "Pearson correlation coefficients - Wing weight")
+view = otv.View(graph)
+
+# %%
+# Spearman coefficients
+# -----------------------
+# We compute here the Spearman :math:`\rho_s` coefficients using the `CorrelationAnalysis`.
+
+# %%
+spearman_correlation = corr_analysis.computeSpearmanCorrelation()
+print(spearman_correlation)
+
+# %%
+graph = ot.SobolIndicesAlgorithm.DrawCorrelationCoefficients(spearman_correlation,
+                                                             inputNames,
+                                                             "Spearman correlation coefficients - Wing weight")
+view = otv.View(graph)
+plt.show()
+
+# %%
+#
+# The different computed correlation estimators show that the variables :math:`S_w, A, N_z, t_c` seem to be the most correlated with the wing weight.

```

Comment:
t_c is anticorrelated, but I suppose you mean this in absolute value. Perhaps it should be explicitely stated?

Reply:
It is now fixed.